python/F3DPythonBindings.cxx

```diff
@@ -258,7 +258,7 @@ PYBIND11_MODULE(pyf3d, module)
     .def("play_interaction", &f3d::interactor::playInteraction, "Play an interaction file")
     .def("record_interaction", &f3d::interactor::recordInteraction, "Record an interaction file")
     .def("start", &f3d::interactor::start, "Start the interactor and the event loop",
-      py::arg("delta_time"), py::arg("user_callback"))
+      py::arg("delta_time") = 1.0 / 30, py::arg("user_callback") = nullptr)
```

Comment:
some test may be updated too I think

Reply:
the existing tests use a lambda to stop, not sure if we can test the default values without that

Reply:
ha right, we need it